ACD-416 (#6)
Actindo-AG/Documentation: RetailSuiteAccounting/Integration/06_ManageFixedBookings.md

[!!User Interface fixed bookings](../UserInterface/02f_FixedBookings.md)  
[!!User Interface select fiscal year](../UserInterface/00a_FiscalYear.md)  
[!!Select a fiscal year](../Operation/01_SelectFiscalYear.md)  

# Manage the fixed bookings

> The *FIXED BOOKINGS* tab in the *Accounting* module enables to automate recurring bookings. The fixed bookings to be automated can be applied when selecting the fiscal year, see [Select the fiscal year](../Operation/01_SelectFiscalYear.md). By doing so, the system will apply automatically the setup fixed bookings in the corresponding fiscal year and journal.

Reviewer: Was sind Festbuchungen überhaupt?

> To be able to create fixed bookings, an existing batch must be selected or a new one must be created first.

Reviewer: gehört unter Create a fixed booking batch

## Create a fixed booking batch

You can create a fixed booking batch that is not yet available in the system.

#### Prerequisites

- The accounting is configured via the configuration wizard, see [Run the accounting wizard](./01_RunAccountingWizard.md).
- A valid fiscal year is created, see [Create a fiscal year](./04_ManageFiscalYear.md#create-a-fiscal-year).

#### Procedure

*Accounting > Select the fiscal year > Settings > Tab FIXED BOOKINGS*

![Create a fixed booking batch](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/CreateFixedBookingBatch.png "[Create a fixed booking batch]")

1. Click the *Fixed booking batch* drop-down list and select the *New batch* option.

>   > [Info] Per default, the drop-down list displays the *New batch* option.

Reviewer: ```suggestion
  > [Info] By default, the *New batch* option is preselected in the drop-down list.
```

2. Enter a batch name in the *Name* field.  

3. Click the [CREATE] button.   

> A new window opens where the new fixed bookings can be registered.

Reviewer: ```suggestion
The *Fixed bookings &ndash; Batch* window is displayed.
```
Bild einfügen

Nächster Schritt: For the next steps to create a fixed booking, follow the procedure [Create a fixed booking]/#create-fixed-booking).

## Delete a fixed booking batch

You can delete an existing fixed booking batch if it is no longer applicable.

#### Prerequisites

- The accounting is configured via the configuration wizard, see [Run the accounting wizard](./01_RunAccountingWizard.md).

> - A valid fiscal year is created, see [Create a fiscal year](./04_ManageFiscalYear.md#create-a-fiscal-year).

Reviewer: Ist das wirklich eine Voraussetzung?

> - A fixed booking batch is created, see [Create a fixed booking batch](#create-a-fixed-booking-batch).

Reviewer: ```suggestion
- At least one fixed booking batch is created, see [Create a fixed booking batch](#create-a-fixed-booking-batch).
```

#### Procedure

*Accounting > Select the fiscal year > Settings > Tab FIXED BOOKINGS*

![Delete an fixed booking batch](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/DeleteFixedBookingBatch.png "[Delete a fixed booking batch]")

> 1. Click the *Fixed booking batch* drop-down list and select the fixed booking batch to be deleted.   

Reviewer: Wenn ich das mache, öffnet sich das Fixed booking &ndash; Batch Fenster. Sollte hier auch entsprechend beschrieben werden.

>   > [Warning] Be aware that the existing fixed booking batch will be deleted permanently. The bookings contained in it, however, will not be deleted from the system and will continue to be displayed in the *Bookings* tab.

Reviewer: nach schritt 2

>   > [Warning] Be aware that the existing fixed booking batch will be deleted permanently. The bookings contained in it, however, will not be deleted from the system and will continue to be displayed in the *Bookings* tab.

Reviewer: ```suggestion
  > [Warning] Be aware that the selected fixed booking batch will be deleted permanently. The bookings contained in it, however, will not be deleted but are still displayed in the *Bookings* tab.
```

2. Click the [DELETE] button.  

> A window comes up asking for confirmation. The selected fixed booking batch is deleted.

Reviewer: ```suggestion
A confirmation window is displayed. 
```
Hier fehlt der Schritt, dass das Löschen bestätigt werden muss, damit die Festbuchung wirklich gelöscht wird.

>   > [Info] You may need to refresh the page with the F5 key to see the changes in the Fixed booking batch drop-down list.

Reviewer: s.o.

> ## Create a fixed booking

Reviewer: Ich glaube, ich würde das umstellen und einen Header ## Create fixed bookings machen mit den zwei procedures ### Create a fixed booking batch und ### Add fixed booking
Ein Stapel ohne Festbuchungen ist zu nichts zu gebrauchen und eine Festbuchung ohne Stapel funktioniert nicht. Daher sollte der Zusammenhang sichtbar gemacht werden

You add fixed bookings to an existing or a newly created fixed booking batch.

#### Prerequisites

- The accounting is configured via the configuration wizard, see [Run the accounting wizard](./01_RunAccountingWizard.md).

> - A valid fiscal year is created, see [Create a fiscal year](./04_ManageFiscalYear.md#create-a-fiscal-year).

Reviewer: Istdas wirkliche eine Voraussetzung?

> - A fixed booking batch is created, see [Create a fixed booking batch](#create-a-fixed-booking-batch).

Reviewer: ```suggestion
- At least one fixed booking batch is created, see [Create a fixed booking batch](#create-a-fixed-booking-batch).
```

#### Procedure

*Accounting > Select the fiscal year > Settings > Tab FIXED BOOKINGS*

> ![Select a fixed booking batch](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/EditFixedBookingBatch.png "[Select a fixed booking batch]")

Reviewer: Unterschied zu Screenshot Delete a fixed booking batch?

> 1. Click the *Fixed booking batch* drop-down list and select the fixed booking batch where the fixed booking must be created.   

Reviewer: ```suggestion
1. Click the *Fixed booking batch* drop-down list and select the fixed booking batch to which you want to add a fixed booking.   
```

> The fixed booking window opens automatically. If it does not, click the [EDIT] button.

Reviewer: ```suggestion
The *Fixed booking &ndash; Batch* window is displayed. 

  > [Info] If the window is not automatically displayed, click the [EDIT] button to display it.
```

>   ![Create a fixed booking](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/CreateFixedBooking.png "[Create a fixed booking]")

Reviewer: Warum hast du das Fenster abgeschnitten?

> 2. Enter a value in the *Turnover* field.

Reviewer: a value? geht das auch etwas genauer?

>   > [Info] The booking numbers must be sequential and are added by the system automatically.

Reviewer: Die Info passt hier nicht hin - wozu soll das gehören?

3. Enter a contra account number in the *Contra account* field.

>   > [Info] When you start typing in an account number, a context menu is displayed. You can also select the appropriate option from the menu.

Reviewer: ```suggestion
  > [Info] After you have entered at least two numbers in the field, a list below the field displays all accounts that match the entry. You can also select the corresponding account in the list.
```

> 4. Enter the date for the fixed booking in the *Date* field. Date format must be TTMMJJ or TT.MM.JJ.

Reviewer: ```suggestion
4. Enter the date for the fixed booking in the *Date* field. The valid date format is *TTMMJJ* or *TT.MM.JJ*.
```

5. Enter an account number in the *Account* field.

> 

Reviewer: > [Info] After you have entered at least two numbers in the field, a list below the field displays all accounts that match the entry. You can also select the corresponding account in the list.

> 

Reviewer: ```suggestion
 > [Info] After you have entered at least two numbers in the field, a list below the field displays all accounts that match the entry. You can also select the corresponding account in the list.
```

>   > [Info] Note that the fields *Contra account*, *Date* and *Account* must be filled out to be able to create a new fixed booking.

Reviewer: ```suggestion
  > [Info] Note that the fields *Contra account*, *Date* and *Account* must be filled out to create a fixed booking.
```
Die Info würde ich ganz an den Anfang stellen, also bevor diese Felder ausgefüllt werden - muss nicht auch das Feld Turnover ausgefüllt werden?

> 6. Enter any other relevant fixed booking information in the corresponding fields.

Reviewer: ```suggestion
6. If desired, enter any other relevant fixed booking information in the corresponding fields.
```

7. Click the [BOOK] button.  

> A message in the window upper left corner confirms that the booking is registered. The newly created fixed booking is displayed in the fixed booking batch.

Reviewer: ```suggestion
The fixed booking is booked. The notice *Booked* is displayed in the upper left corner of the window. The created fixed booking is displayed in the *Fixed bookings &ndash; Batch* window.
```

>   ![Fixed booking created](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/FixedBookingCreated.png "[Fixed booking created]")

Reviewer: s.o. Fenster abgeschnitten

8. Click the [SAVE] button.   

> The booking is now saved in the batch.

Reviewer: ```suggestion
The booking is saved in the batch.
```

>  > [Info] If you click the [CANCEL] button, the fixed booking will not be saved.

Reviewer: ```suggestion
 > [Info] If you click the [CANCEL] button, the fixed booking will be rejected.
```

## Edit a fixed booking

> You can edit existing fixed bookings registered in a fixed booking batch.

Reviewer: ```suggestion
You can edit fixed bookings registered in a fixed booking batch.
```

### Prerequisites

- The accounting is configured via the configuration wizard, see [Run the accounting wizard](./01_RunAccountingWizard.md).

> - A valid fiscal year is created, see [Create a fiscal year](./04_ManageFiscalYear.md#create-a-fiscal-year).

Reviewer: Ist das wirklich eine Voraussetzung?

- A fixed booking batch is created, see [Create a fixed booking batch](#create-a-fixed-booking-batch).
- A fixed booking is created, see [Create a fixed booking](#create-a-fixed-booking).

### Procedure

*Accounting > Select the fiscal year > Settings > Tab FIXED BOOKINGS*

> ![Select a fixed booking batch](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/EditFixedBookingBatch.png "[Select a fixed booking batch]")

Reviewer: s.o.

> 1. Click the *Fixed booking batch* drop-down list and select the fixed booking batch where the fixed booking to be edited is registered.   

Reviewer: ```suggestion
1. Click the *Fixed booking batch* drop-down list and select the fixed booking batch of the fixed booking to be edited.   
```

> The fixed booking window opens automatically. If it does not, click the [EDIT] button.

Reviewer: wie oben

> 2. Click on the existing fixed booking to be edited.

Reviewer: ```suggestion
2. Click the fixed booking to be edited.
```

> The existing fixed booking details are displayed in the fields.

Reviewer: ```suggestion
The details of the selected fixed booking are displayed in the fields.
```

>   ![Edit a fixed booking](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/EditFixedBooking.png "[Edit a fixed booking]")

Reviewer: s.o.

3. Make any necessary changes in the fields.

4. Click the [BOOK] button.  

> The edited fixed booking is added to the fixed booking batch with a new sequential number.

Reviewer: ```suggestion
A new fixed booking with the indicated modifications is added to the fixed booking batch with a new sequential number.
```

>   > [Info] The old fixed booking can be now deleted from the batch. This two-step editing process prevents inadvertently overwriting.

Reviewer: Diese Info würde ich umformulieren und als Einleitung für die Procedure verwenden. Das Löschen der alten Festbuchung muss hier noch schrittweise beschrieben werden, da es Teil des Bearbeitungsprozesses ist.

5. Click the [SAVE] button.   

> The edited fixed booking is now saved in the batch.  

Reviewer: ```suggestion
All changes to the fixed bookings are saved.  
```

>   > [Info] If you click the [CANCEL] button, the changes will not be saved.

Reviewer: ```suggestion
  > [Info] If you click the [CANCEL] button, the changes will be rejected.
```

## Delete a fixed booking

> You can delete an existing fixed booking in a fixed booking batch.

Reviewer: ```suggestion
You can delete a fixed booking in a fixed booking batch.
```

#### Prerequisites

- The accounting is configured via the configuration wizard, see [Run the accounting wizard](./01_RunAccountingWizard.md).

> - A valid fiscal year is created, see [Create a fiscal year](./04_ManageFiscalYear.md#create-a-fiscal-year).

Reviewer: Ist das wirklich eine Voraussetzung?

- A fixed booking batch is created, see [Create a fixed booking batch](#create-a-fixed-booking-batch).
- A fixed booking is created, see [Create a fixed booking](#create-a-fixed-booking).

#### Procedure

*Accounting > Select the fiscal year > Settings > Tab FIXED BOOKINGS*

> ![Select an existing batch](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/EditFixedBookingBatch.png "[Select an existing batch]")

Reviewer: Unterschied zu anderen Screenshots? Wiederverwendung!

> 1. Click the *Fixed booking batch* drop-down list and select the batch where the fixed booking to be deleted is registered.    

Reviewer: ```suggestion
1. Click the *Fixed booking batch* drop-down list and select the batch of the fixed booking to be deleted.    
```

> The fixed booking window opens automatically. If it does not, click the [EDIT] button.

Reviewer: wie oben

>   ![Delete a fixed booking](../../Assets/Screenshots/RetailSuiteAccounting/Settings/FixedBookings/DeleteFixedBooking.png "[Delete a fixed booking]")

Reviewer: s.o. abgeschnitten, Wiederverwendung

> 2. Click on the existing fixed booking to be deleted.

Reviewer: ```suggestion
2. Click the fixed booking to be deleted.
```

> The existing booking details are displayed in the fields.

Reviewer: ```suggestion
The details of the selected fixed booking are displayed in the fields.
```

>   > [Warning] Be aware that the existing fixed booking will be deleted permanently from the fixed bookings batch. The fixed booking itself, however, will not be deleted from the system and will continue to be displayed in the *Bookings* tab.

Reviewer: ```suggestion
  > [Warning] Be aware that the fixed booking will be deleted permanently from the fixed bookings batch. The fixed booking itself, however, will not be deleted from the system and will continue to be displayed in the *Bookings* tab.
```
Das widerspricht sich irgendwie... Wird die Festbuchung nun gelöscht oder nicht?

3. Click the [DELETE] button.  

> A window comes up asking for confirmation. The selected fixed booking is deleted from the batch.

Reviewer: s.o.

4. Click the [SAVE] button.   

> The deletion is now saved in the batch.

Reviewer: ```suggestion
The fixed booking deletion is saved.
```

>  > [Info] If you click the [CANCEL] button, the deletion will not be saved.

Reviewer: ```suggestion
 > [Info] If you click the [CANCEL] button, the deletion will be rejected.
```
